docs/content/doc/features/comparison.en-us.md

```diff
@@ -50,7 +50,8 @@ _Symbols used in table:_
 | Repository Tokens with write rights | ✓                                                  | ✘    | ✓         | ✓         | ✓         | ✓              | ✓            |
 | Built-in Container Registry         | [✘](https://github.com/go-gitea/gitea/issues/2316) | ✘    | ✘         | ✓         | ✓         | ✘              | ✘            |
 | External git mirroring              | ✓                                                  | ✓    | ✘         | ✘         | ✓         | ✓              | ✓            |
-| FIDO U2F (2FA)                      | ✓                                                  | ✘    | ✓         | ✓         | ✓         | ✓              | ✘            |
+| FIDO U2F (2FA)(Dropped by browsers) | ✘                                                  | ✘    | ✓         | ✓         | ✓         | ✓              | ✘            |
+| WebAuthn (2FA)                      | ✓                                                  | ✘    | ✓         | ✓         | ✓         | ✓              | ✘            |
```

Comment:
unfortunately the webauthn PR removed U2F support and replaced it with webauthn. We provide a fallback for keys that were registered with U2F but you cannot register a U2F key even if you have an old browser that still supports this.

I vote replace FIDO U2F with WebAuthn in this table.

Reply:
Yes, let's remove the legacy U2F line in the table.

Reply:
Done.